python/ray/util/iter.py

```diff
@@ -680,6 +711,44 @@ def add_wait_hooks(it):
             self.local_transforms + [apply_foreach],
             name=self.name + ".for_each()")
 
+    def for_each_concur(self,
+                        fn: Callable[[T], U],
+                        max_concur=2,
+                        resources=None) -> "LocalIterator[U]":
+        if resources is None:
+            resources = {}
```

Comment:
Can you document?

Reply:
All of the local iterator functions share documentation with their respective parallel iterator versions